lib/utilities/reportHandler.js

```diff
@@ -73,6 +74,56 @@ function getSystemStats() {
     };
 }
 
+function getCRRStats(log, cb) {
+    log.debug('getting CRR stats', { method: 'getCRRStats' });
+    // TODO: Reuse metrics code from Backbeat by moving it to Arsenal instead of
+    // making an HTTP request to the Backbeat metrics route.
+    const params = { url: 'http://localhost:8900/_/metrics/crr/all' };
```

Comment:
even if it's temporary, having a configuration would be easier because we currently need to support 4 different stacks (swarm, kube, federation, sandboxes) and not all of them are able to reach backbeat at `localhost:8900`.